db/data_migrations/20240508102256_update_rejected_offender_sar_data_to_warehouse.rb

```diff
@@ -0,0 +1,21 @@
+class UpdateRejectedOffenderSARDataToWarehouse < ActiveRecord::DataMigration
+  def up
+    Case::Base.offender_sar.where("properties ->> 'case_originally_rejected' = 'true'").find_each do |k|
+      k.update_attribute(:rejected_reasons, Warehouse::CaseReport.rejected_reasons_selection(k)) unless k.readonly? # rubocop:disable Rails/SkipsModelValidations
+    end
+
+    execute <<-SQL
```

Comment:
I don't think this is necessary, because all the rejected cases will get updated anyway with the change to rejected_reasons. That should also be true for other_rejected_reason.